source/state/vault/index.ts

```diff
@@ -32,18 +24,15 @@ const initialState: IVaultState = {
     [KeyringNetwork.Constellation]: DAG_NETWORK.main.id,
     [KeyringNetwork.Ethereum]: ETH_NETWORK.mainnet.id,
   },
-  version: '2.1.1' ,
+  version: '2.1.1',
```

Comment:
Should this version number be bumped?

Reply:
I'm not actually sure how we're using that. Frank was the last one to update it.